### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by rishavquventix**

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -13,5 +19,101 @@ app.use(express.json());
...
+            if(ext==1){
+                const extension  =  words[ext];
+            cb(null, file.fieldname + "-" + Date.now()+"."+extension); 
+            }
+            
+        }
+
+    } 
+})
+
```
Comment (officialanurag) [inline app.js:78]: Use const. do proper indetation, remove extra space

File: app.js [hunk 2]
```diff
@@ -13,5 +19,101 @@ app.use(express.json());
...
+    }, 
+    filename: function (req, file, cb) { 
+
+        const words = file.originalname.split('.');
+
+        for (var ext in words) 
+        {
+            if(ext==1){
+                const extension  =  words[ext];
+            cb(null, file.fieldname + "-" + Date.now()+"."+extension);
```
Comment (officialanurag) [inline app.js:71]: Use literal templating instead of concatenation

File: app.js [hunk 3]
```diff
@@ -13,5 +19,101 @@ app.use(express.json());
...
+            }
+            
+        }
+
+    } 
+})
+
+const maxSize = 10* 1000 * 1000; 
+    
+var upload = multer({
```
Comment (officialanurag) [inline app.js:81]: Use const.

File: app.js [hunk 4]
```diff
@@ -13,5 +19,101 @@ app.use(express.json());
...
+const maxSize = 10* 1000 * 1000; 
+    
+var upload = multer({  
+    storage: storage, 
+    limits: { fileSize: maxSize }, 
+    fileFilter: function (req, file, cb){ 
+        var filetypes = /mp4|mp3/; 
+        var mimetype = filetypes.test(file.mimetype); 
+        var extname = filetypes.test(path.extname(file.originalname).toLowerCase()); 
+
```
Comment (officialanurag) [inline app.js:88]: user const if they are not modified any where else user let.

File: app.js [hunk 5]
```diff
@@ -13,5 +19,101 @@ app.use(express.json());
...
+        if (!mimetype && !extname) { 
+            return cb(null, true);
+        }
+
+        cb("Error: File upload not supports the following filetypes - " + filetypes); 
+        }  
+
+}).single("myfile");        
+
+app.get("/",  verify ,function(req,res){
```
Comment (officialanurag) [inline app.js:101]: Literal templating

File: app.js [hunk 6]
```diff
@@ -13,5 +19,101 @@ app.use(express.json());
...
+    upload(req,res,function(err) { 
+
+        if(err) {  
+            res.send(err) 
+        } 
+        else { 
+            res.send("Success, File uploaded!") 
+        } 
+    }) 
+})
```
Comment (officialanurag) [inline app.js:116]: Do proper indetation. remove extra space.

File: app.js (hunk 1 above)
Comment (rishavquventix) [reply app.js]: Done Sir

File: app.js (hunk 2 above)
Comment (rishavquventix) [reply app.js]: Done Sir

File: app.js (hunk 3 above)
Comment (rishavquventix) [reply app.js]: Done Sir

File: app.js (hunk 4 above)
Comment (rishavquventix) [reply app.js]: Done Sir

File: app.js (hunk 5 above)
Comment (rishavquventix) [reply app.js]: Done Sir

File: app.js (hunk 6 above)
Comment (rishavquventix) [reply app.js]: Done Sir

**Pull request merged**